actors/miner/src/types.rs

```diff
@@ -467,7 +467,7 @@ pub struct ProveReplicaUpdatesParams {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
-pub struct ReplicaUpdate2 {
+pub struct ReplicaUpdateDeprecated {
     pub sector_number: SectorNumber,
```

Comment:
The bodies of these structs now exist only so that the method can fail with "no such method" _after_ deserializing into them, rather than a serialisation error. I think either of two options would be better:
- delete the bodies of these structs and use the `[raw]` syntax in dispatch to defer deserialisation of the params
- remove the explicit method handler for the deprecated method and let it fail with "no such method" without a nice error message, like the deprecated `PreCommitSectorBatch`. Remove these types entirely.

The second is simpler with very minor drawback.